src/TLuaInterpreter.cpp

```diff
@@ -1623,26 +1623,80 @@ int TLuaInterpreter::feedTriggers(lua_State* L)
 {
     Host& host = getHostFromLua(L);
 
-    std::string text;
     if (!lua_isstring(L, 1)) {
         lua_pushfstring(L,
                         "feedTriggers: bad argument #1 type (imitation MUD server text as string\n"
                         "expected, got %s!)",
                         luaL_typename(L, 1));
         return lua_error(L);
-    } else {
-        text = lua_tostring(L, 1);
     }
 
-    QString previousEncoding = host.mTelnet.getEncoding();
-    if (previousEncoding.isEmpty()) {
-        previousEncoding = QStringLiteral("ASCII");
+    bool dataIsUtf8Encoded = false;
+    if (lua_gettop(L) > 1) {
+        if (!lua_isboolean(L, 2)) {
+            lua_pushfstring(L,
+                            "feedTriggers: bad argument #2 type (dataIsUtf8Encoded as boolean {defaults to\n"
+                            "false if omitted} is optional, got %s!)",
```

Comment:
We've discussed this previously and decided not to repeat the API. Keep the message as short and to the point as necessary, don't add extra unrelated information: error messages are very hard to understand already, adding unrelated information will make it even harder.

Those who do understand them will not see much value in it either.

Reply:
Well, I have shortened it a bit - but

Reply:
Okay, will remove that detail in next commit.